### PR TITLE
feat: rewrite Iterate with JavaScript While Loops challenge

### LINKED ISSUE #37694: (curriculum): Iterate with JavaScript While Loops
I noticed [this](https://github.com/freeCodeCamp/freeCodeCamp/pull/37683#pullrequestreview-311911070) problem when reviewing that PR.

The example in the challenge description, is the exact code that instructions say to implement...

Screen Shot 2019-11-05 at 12 58 24 PM

[Link to the challenge](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops) 

We could probably change either the example or the coding part to make them a little different.

Maybe just something as simple as changing the instructions to...
`Push the numbers 5 through 9 to myArray using a while loop.`

or perhaps slightly more difficult...
`Push the numbers 5 through 0, in that order and including both the 5 and 0, to myArray using a while loop`

### PR DESCRIPTION
**Pull request by PartyLich**
Previous example in the challenge description was the exact code
that challenge instructions say to implement. Change the challenge to
differentiate from the example.

Closes #37694

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Let's try getting a while loop to work by pushing values to an array.
 
 ## Instructions
 <section id='instructions'>
-Push the numbers 0 through 4 to <code>myArray</code> using a <code>while</code> loop.
+Push the numbers 5 through 0, in that order and including both the 5 and 0, to <code>myArray</code> using a <code>while</code> loop.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md]:
```suggestion
Add the numbers 5 through 0 (inclusive) in descending order to <code>myArray</code> using a <code>while</code> loop.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my suggestion for wording change.

**Comment by PartyLich** [comment]
Wording suggestion is definitely an improvement. I was a bit lazy in grabbing the suggested text directly from the issue. Used the direct commit via github feature for expediency, hopefully it hasn't done anything undesirable or unexpected.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@PartyLich Thanks for making the changes.  Everything looks good to me know.  I will leave it open for another mod to review before merging.

**Review by Manish-Giri — APPROVED** [review]
LGTM. Thank You for your contribution.

**Pull request merged**